src/scenic/core/object_types.py

```diff
@@ -671,9 +660,10 @@ def __getattr__(self, attr):
 class OrientedPoint(Point):
     """The Scenic class ``OrientedPoint``.
 
-    The default mutator for `OrientedPoint` adds Gaussian noise to ``heading``
-    with a standard deviation given by the ``headingStdDev`` property, then
-    applies the mutator for `Point`.
+    The default mutator for `OrientedPoint` adds Gaussian noise to ``yaw`` while
+    leaving ``pitch`` and ``roll`` unchanged, using the three standard deviations
+    (for yaw/pitch/roll respectively) given by the  ``orientationStdDev`` property.
```

Comment:
This doesn't really make sense -- why do you need 3 different stddev values if you're only modifying `yaw`? I would say it adds Gaussian noise to all 3 properties according to `orientationStdDev`, but that the default stddevs for pitch and roll are zero so that by default only `yaw` is mutated.